src/main/java/com/example/naitei19javaecommerce/controller/admin/InvoiceController.java

```diff
@@ -0,0 +1,46 @@
+package com.example.naitei19javaecommerce.controller.admin;
+
+import com.example.naitei19javaecommerce.dto.PaymentHistoryResponse;
+import com.example.naitei19javaecommerce.service.InvoiceService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+import java.math.BigDecimal;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.List;
+
+@Controller
+@RequestMapping("/admin")
+public class InvoiceController {
+    private final InvoiceService invoiceService;
+
+    @Autowired
+    public InvoiceController(InvoiceService invoiceService ) {
+        this.invoiceService = invoiceService;
+    }
+
+    @GetMapping("/payment")
```

Comment:
mới check thì cái ni nên là / thay vì `payment` nha 
```suggestion
    @GetMapping("/")
```

Reply:
Cái này chưa fix nè
admi/invoices thôi, thêm payments làm j gái
theo chuẩn restful đi gái

fix ở pull sau